cmd/api/http/handlers.go

```diff
@@ -15,7 +15,7 @@ import (
 	"github.com/google/uuid"
 )
 
-var timeout = time.Duration(1) * time.Millisecond
+var timeout = time.Duration(30) * time.Millisecond
```

Comment:
Set 30 mS to test timeout with Postman. I think we could set it with 5 seconds to normal operation.

Reply:
This should be read from the environment as configuration.